src/systems/si/include/mp_units/systems/si/units.h

```diff
@@ -41,6 +41,11 @@ inline constexpr struct mole : named_unit<"mol", kind_of<isq::amount_of_substanc
 inline constexpr struct candela : named_unit<"cd", kind_of<isq::luminous_intensity>> {} candela;
 
 // derived named units
+#ifdef pascal
+#pragma push_macro("pascal")
+#undef pascal
+#define UNITS_REDEFINE_PASCAL
+#endif
```

Comment:
@JohelEGP do you think it should be guarded with some compiler ID or global like that?

Reply:
I have no strong opinions right now.
What is done elsewhere?
Were the other instances fixed by removing the space between `operator""` and `𝘴𝘺𝘮𝘣𝘰𝘭`?
If so, what was done before that, could it be done again?

Reply:
This is not about UDL but a concrete identifier of a type. We had `pascal` before in V1 as a unit of `pressure`, but maybe no one used it on MinGW before.

Reply:
> Anyway, it would be probably better to put `#ifdefs` only around `pascal` definition rather than the entire block of units.

Can do, I didn't want to disrupt the definitions.


> @JohelEGP do you think it should be guarded with some compiler ID or global like that?

Under the assumption that only the win api headers define `pascal` I don't think one would need a guard, because the guard is the `#ifdef`. Same goes for `push_macro`, [GCC defines it "_For compatibility with Microsoft Windows compilers_"](https://gcc.gnu.org/onlinedocs/gcc/Push_002fPop-Macro-Pragmas.html), the same would have to be true for all compilers targeting windows.

Reply:
> Can do, I didn't want to disrupt the definitions.

Please do it if you can. The definitions are disturbed anyway as the macro is not disabled for the entire file but only a part of it.